### PR TITLE
auth controller setup

### PR DESCRIPTION
**Pull request by wowKimsohyun17**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/30235)

## WHAT this PR does?
basic auth controller

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by wowKimsohyun17** [comment]
ready

**Comment by wowKimsohyun17** [comment]
ready

File: database/migrations/2020_10_26_011040_modifier_attribute_users_table.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('users', function (Blueprint $table) {
+            //
+        });
```
Comment (huyenttm-0901) [inline database/migrations/2020_10_26_011040_modifier_attribute_users_table.php]: viết hàm down()

**Comment by wowKimsohyun17** [comment]
ready

**Pull request merged**